proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -190,6 +198,21 @@ message Method {
   string description = 2;
   MethodRequestResponse request = 3;
   MethodRequestResponse response = 4;
+  MethodOptions method_options = 5;
+}
+
+// MethodOptions provide information for the documentation of options for a method.
+message MethodOptions {
+  reserved "uninterpreted_options";
+  bool deprecated = 1;
+  IdempotencyLevel idempotency_level = 2;
+  repeated string custom_options = 1000;
```

Comment:
For the custom options, is there a reason why we are matching this to descriptor proto and using the field value `1000`? My understanding is for the file descriptor, specifically, 1000+ is for extensions. This should not be a concern here, right?

Reply:
Correct. I was initially thinking it'd be good to keep the field numbers similar to the descriptor proto, but it would rather create unnecessary confusion instead. Changed them.